CHANGELOG.md

```diff
@@ -1,19 +1,29 @@
+### Migrating to UniFFI 0.23
+
+- Update your `Cargo.toml` file to only depend on the `uniffi` crate.  Follow the directions from the [Prerequisites section of the manual](https://mozilla.github.io/uniffi-rs/tutorial/Prerequisites.html)
+- Create a `uniffi-bindgen` binary for your project.  Follow the directions from the [Foreign language bindings section of the manual](https://mozilla.github.io/uniffi-rs/tutorial/foreign_language_bindings.html).
+- Uninstall the system-wide `uniffi_bindgen`: `cargo uninstall uniffi_bindgen`.  (Not strictly necessary, but you won't be using it anymore).
+
```

Comment:
@mhammond suggested making it sticky for a while, in case a user misses several releases and then tries to upgrade.  This makes sense for about 6-12 months, then we can move it back down.  That said, I don't have strong feelings about this one.

Reply:
Ah, didn't see that. Yeah, I'm fine with that.